ecommers/lib/ui/widgets/button/primary_button_widget.dart

```diff
@@ -6,13 +6,16 @@ class PrimaryButtonWidget extends ButtonBaseWidget {
   PrimaryButtonWidget({
     @required String text,
     @required Function() onPressedFunction,
+    Color backgroundColor = BrandingColors.primary,
```

Comment:
do you really need these changes? 
Please have a look at the SecondaryButtonWidget